### ISSUE TITLE
UPDATE CONTENT adding ORestricted properties

### ISSUE THREAD

**Issue by sinfex** [orientechnologies/orientdb#5564]
**UPDATE CONTENT** seems to be adding record security properties to records even when the target records do not belong to the ORestricted class.

```
orientdb {db=test}> CREATE VERTEX V
Created vertex 'V#9:0 v1' in 0.013000 sec(s).

orientdb {db=test}> UPDATE #9:0 MERGE {"a":1}
Updated record(s) '1' in 0.003000 sec(s).

orientdb {db=test}> SELECT FROM #9:0
----+----+------+----
#   |@RID|@CLASS|a
----+----+------+----
0   |#9:0|V     |1
----+----+------+----

orientdb {db=test}> UPDATE #9:0 CONTENT {"a":2}
Updated record(s) '1' in 0.007000 sec(s).

orientdb {db=test}> SELECT FROM #9:1
----+----+------+----+------------+------------+----------+------
#   |@RID|@CLASS|a   |_allowUpdate|_allowDelete|_allowRead|_allow
----+----+------+----+------------+------------+----------+------
0   |#9:0|V     |2   |null        |null        |null      |null
----+----+------+----+------------+------------+----------+------
```

Version: OrientDB Server v2.1.8 (build 2.1.x@r; 2015-12-17 10:18:05+0000)

**Comment by luigidellaquila** [comment]
Hi @sinfex 

thanks for reporting, I could reproduce the issue locally. I flagged it as a bug and I'll work on it as soon as possible

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Fixed on develop branch, the fix will be released in v 2.2.1

Thanks

Luigi
